.github/workflows/build-test.yaml

```diff
@@ -54,6 +54,29 @@ jobs:
           path: coverage
           retention-days: 5
 
+  playwright:
+    needs: [yarn_cache]
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Use Node.js 14.x
+        uses: actions/setup-node@v3
+        with:
+          node-version: 14.x
+          cache: 'yarn'
+      - run: yarn install --frozen-lockfile
+      - run: yarn start &
+      - run: yarn install --frozen-lockfile
```

Comment:
this one was run in ~L67

Reply:
Nope, I'll try to make it clearer that this runs in a different folder (because I isolated playwright into its own folder)

Reply:
Hm, CI say it only takes 1 second to install the subfolder. Un-isolate?

Reply:
I'm fine with both approaches.

Reply:
Kept isolated